tensorflow/python/keras/layers/normalization_test.py

```diff
@@ -216,6 +216,25 @@ def test_step(x, y):
     if context.executing_eagerly():
       self.assertAlmostEqual(test_loss.numpy(), train_loss.numpy())
 
+  @keras_parameterized.run_all_keras_modes
+  def test_batchnorm_ignore_masked_values(self):
+    padded_data = np.array([[[1, 5], [2, 5], [0, 0], [0, 0]] for _ in range(10)], dtype='float32')  # Pad value of 0
+    data = np.array([[[1, 5], [2, 5]] for _ in range(10)], dtype='float32')
+
+    input = tf.keras.layers.Input((None, 2))
+    masked = tf.keras.layers.Masking()(input)
+    normed = BatchNormalization(momentum=0.0)(masked)
+    model = tf.keras.models.Model(input, normed)
+    model.compile('rmsprop', 'mse',
+                  run_eagerly=testing_utils.should_run_eagerly())
+
+    model.fit(x=padded_data, y=padded_data, batch_size=10, epochs=5)
+    moments_mask = model.layers[2].get_weights()
+    model.fit(x=data, y=data, batch_size=10, epochs=5)
+    moments = model.layers[2].get_weights()
+    self.assertEqual(moments_mask[2][0], moments[2][0]) # moving average mean comparison
```

Comment:
Assume the moments_mask[2] is the moving average mean here, why we only compare the first element which is [0] here?